### DISCUSSION TITLE
What to name this Wallet Developers Community?

### DISCUSSION THREAD

**Discussion by ChristopherA** [BlockchainCommons/Gordian-Developer-Community#9] [Ideas]
What should we name this community? Continue just calling it "AirgappedSigning"? Is there a better name?

-- Christopher Allen

**Comment by msgilligan** [comment]
This just popped into my head: "Corpus Callosum" (the connection between the two parts)

**Comment by ChristopherA** [comment]
Related, this is what I have as our mission in the [README.md](https://github.com/BlockchainCommons/AirgappedSigning/blob/master/README.md). Suggestions and/or PRs welcome.

> Airgapped Signing is a community of cryptocurrency wallet developers, sharing best practices and interoperability for airgapped QR code communications between multiple vendors on multiple platforms. These best practices include support for hot, cold and multisig cryptographic wallets, research & proposals for data formats and protocols, security and UX for multisig work flows, and other key management and signing issues and topics.

**Comment by wolfmcnally** [comment]
Could clarify the name a bit by calling it "Airgapped Wallet Developers".

**Comment by ChristopherA** [comment]
For now I have renamed this community "Airgapped-Wallet-Community". You should be redirected here if you use the old URL, but please change this URL in links, and in particular to tis discussions area https://github.com/BlockchainCommons/Airgapped-Wallet-Community/discussions

Thanks!
